test/README.md

```diff
@@ -51,50 +51,54 @@ More details in config/README.md.
 
 All FIT tests can be run from the wrapper 'run_tests.py':
 
-    usage: run_tests.py [-h] [-test TEST] [-config CONFIG] [-group GROUP]
-                        [-stack STACK] [-ora ORA] [-version VERSION]
-                        [-template TEMPLATE] [-xunit] [-numvms NUMVMS] [-list]
-                        [-sku SKU] [-obmmac OBMMAC | -nodeid NODEID]
-                        [-http | -https] [-port PORT] [-v V]
-
-    Command Help
-
-    optional arguments:
-      -h, --help          show this help message and exit
-      -test TEST          test to execute, default: tests/
-      -config CONFIG      config file location, default: config
-      -group GROUP        test group to execute: 'smoke', 'regression',
-                          'extended', default: 'all'
-      -stack STACK        stack label (test bed), overrides -ora
-      -ora ORA            OnRack/RackHD appliance IP address or hostname, default:
-                          localhost
-      -version VERSION    OnRack package install version, example:onrack-
-                          release-0.3.0, default: onrack-devel
-      -template TEMPLATE  path or URL link to OVA template or OnRack OVA
-      -xunit              generates xUnit XML report files
-      -numvms NUMVMS      number of virtual machines for deployment on specified
-                          stack
-      -list               generates test list only
-      -sku SKU            node SKU name, example: Quanta-T41, default=all
-      -obmmac OBMMAC      node OBM MAC address, example:00:1e:67:b1:d5:64
-      -nodeid NODEID      node identifier string of a discovered node, example:
-                          56ddcf9a8eff16614e79ec74
-      -http               forces the tests to utilize the http API protocol
-      -https              forces the tests to utilize the https API protocol
-      -port PORT          API port number override, default from
-                          install_config.json
-      -v V                Verbosity level of console output, default=1, Built Ins:
-                          0: No debug, 2: User script output, 4: rest calls and
-                          status info, 6: other common calls (ipmi, ssh), 9: all
-                          the rest
+### --help output
+        usage: run_tests.py [-h] [-test TEST] [-config CONFIG] [-group GROUP]
```

Comment:
it wasn't doing the markdown right before... looked fine in a text editor, but was messed up elsewhere... (the ### didn't impact it)

However, I did realize that I forgot to recapture the updated output after the most recent changes. Doing that now.